behaviour/graql/language/insert.feature

```diff
@@ -1474,6 +1578,76 @@ Feature: Graql Insert Query
       | BRO |
 
 
+  Scenario: inserting a new instance type on an existing instance is fine if it is already of that type
```

Comment:
this title doesn't read super clearly

Reply:
Updated to `Scenario: re-inserting a matched instance as an unrelated type throws an error`